### ISSUE TITLE
Easier way to check units, useful for threshold in watershed

### ISSUE THREAD

**Issue by bw4sz** [brycefrank/pyfor#14]
Hey bryce - this is looking really great! Amazing job. I just came back to it for a way to pass watershed clusters to my deep learning tree classifier (still in progress). One thing i noticed is that its not immediately easy to know the map units of the vertical dimension. It might be useful to have a summary method for the cloud object that prints some essentials. 

Similar to

https://rdrr.io/cran/lidR/man/summary.LAS.html

That way I know what units to input to the threshold tolerance for watershed.

**Comment by brycefrank** [comment]
This should be straightforward, I will implement in 0.2.2 when I get the chance.

Thanks for the suggestion, Ben.

**Comment by bw4sz** [comment]
yup. Just letting you know, rasterio's planned update is going to break your code. I found it in pre release.

```
pip install --pre rasterio --upgrade
```
Has some kind of problem way deep in the code, looks like something with __gdal_version__ variable. I can't paste it, i've already downgraded. I had grabbed it because i was looking for a way to crop the CHM by the shapely geometry from your geopandas frame. I'm trying to avoid writing the chm to file (big arrays), which fits better with rasterio's current functionality. Soon you will be able to push numpy arrays directly to their file type.

https://rasterio.readthedocs.io/en/latest/topics/memory-files.html

**Comment by brycefrank** [comment]
Thanks for the tip, I will adjust the yaml file to maintain the old version once Travis CI fails.

0Bryce

**Comment by brycefrank** [comment]
Ben,

Just an update - I am working on this (summary function) for `0.3.0`. It is proving a bit tricky with different LAS specifications, but I think I can come up with something reasonable that attempts to parse CRS information from older acquisitions. The new version should be out sometime in the next month or two.

-Bryce
